assets/training/model_management/environments/foundation-model-inference/context/requirements.txt

```diff
@@ -1,19 +1,11 @@
-git+https://github.com/microsoft/DeepSpeed.git
-git+https://github.com/microsoft/DeepSpeed-MII.git@mrwyattii/update-AML-deployment
-
-aiolimiter==1.1.0
-torch==2.0.1
-protobuf==4.23.4
+pandas
```

Comment:
we can pin specific versions

Reply:
should we just use them from here? https://github.com/Azure/azureml-examples/blob/main/sdk/python/foundation-models/system/inference/text-generation/llama-files/docker_env/conda_dependencies.yaml

Reply:
actually, it doesn't have pandas, numpy at all there